src/Microsoft.DotNet.CodeFormatting/Rules/NewLineAboveRule.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.DotNet.CodeFormatting.Rules
     /// <summary>
     /// Ensure there is a blank line above the first using and namespace in the file. 
     /// </summary>
-    [SyntaxRule(SyntaxRuleOrder.NewLineAboveFormattingRule)]
+    [SyntaxRule(SyntaxRuleOrder.NewLineAboveFormattingRule, Skip = true)]
```

Comment:
I'm not a fan of it, but mostly due to when you don't specify a copyright header it'll insert a newline before all your using statements anyway.